tests/testthat/helper_create_ini.R

```diff
@@ -0,0 +1,20 @@
+# Helper functions for tests in test_create_ini.R
+# Matt Nicholson
+# 20 Feb 2020
+
+fetch_hector_ini <- function(scenario) {
+  hector_ini_name <- parse_ini_fname(scenario)
+  hector_ini_path <- system.file('input', hector_ini_name, package='hector')
```

Comment:
The tests were failing for me for awhile before I realized that I needed to install hector. Please add some sort of check to assert that the hector_ini_path is a file that exists.  

You are more familiar with `renv` than me but I was under the impression that  renv could be used to load all of these dependencies. Would that be from running `renv::restore` or `renv::load`?

Reply:
`renv::restore()` (or `Rscript -e 'renv::restore()'`) will begin the installation of packages described in `renv.lock`. More on reproducing `renv` libraries [here](https://rstudio.github.io/renv/articles/renv.html#reproducibility)

Reply:
Okay when I try to run renv::restore() I run into some issues related to `nlme`

```
- 'ld: library not found for -lgfortran'
Error: install of package 'nlme' failed
```

Where are we using `nlme`?

Reply:
Great question....we're not. I think it was installed as a dependency's dependency. I re-initialized the renv library and installed the dependencies via `install.packages(c("devtools", "usethis", "testthat"), dependencies=c("Depends", "Imports"))`, but `nlme` was installed again

Reply:
Hmmm interesting if you check out the reverse dependencies on CRAN https://cran.r-project.org/web/packages/nlme/index.html are we using any of those packages?  Usually I am not that picky about package dependencies so long as it is clear what needs to be installed. If we have to use `nlme` fine but I am not excited about it because it looks like the installation requires multiple steps.

Reply:
An alternative to reading the ini file from the installed Hector would be to pull from the remote branch I've done this with csv files for another project so something like... 
```  readr::read_csv(url("https://raw.githubusercontent.com/JGCRI/CMIP6/master/cmip6_archive_index.csv"))
```

Reply:
readr::read_csv(url("https://raw.githubusercontent.com/JGCRI/CMIP6/master/cmip6_archive_index.csv"))